library/src/scala/tasty/reflect/TreeOps.scala

```diff
@@ -272,8 +272,9 @@ trait TreeOps extends Core {
     /** Scala term selection */
     val Select: SelectModule
     abstract class SelectModule {
+      def unique(qualifier: Term, name: String)(implicit ctx: Context): Select
 
-      // TODO def apply(qualifier: Term, name: String, signature: Option[Signature])(implicit ctx: Context): Select
+      def overloaded(qualifier: Term, name: String, targs: List[Type], args: List[Term])(implicit ctx: Context): Apply
```

Comment:
This should probably be the `apply` as it is the default constructor.

Reply:
Indeed, `apply` will make the syntax shorter in usage. Here, I think there is one advantage of using `overloaded`: remind meta-programmers there is always possibility of overloading, thus the arguments should be provided for overloading resolution. That saves us from documenting the method and justifying the API design.

Reply:
Another thought is that we may resort `apply` for selection with a symbol.